tests/apps/loqus/test_loqusdbapi.py

```diff
@@ -0,0 +1,70 @@
+import subprocess
+
+from cg.apps.loqus import LoqusdbAPI
+
+def test_instatiate(loqus_config):
+    """Test to instantiate a loqusdb api"""
+    ## GIVEN a loqusdb api with some configs
+
+    ## WHEN instantiating a loqusdb api
+    loqusdb = LoqusdbAPI(loqus_config)
+
+    ## THEN assert that the adapter was properly instantiated
+    assert loqusdb.uri == loqus_config['loqusdb']['database']
+    assert loqusdb.db_name == loqus_config['loqusdb']['database_name']
+    assert loqusdb.loqusdb_binary == loqus_config['loqusdb']['binary']
+
+def test_get_case(loqusdbapi, mocker):
```

Comment:
I can't find this mocker in any of the `conftest.py`.

Reply:
It's magic! Doesn't need to be imported, it connects with pytest in some way

Reply:
It's loaded in setup.py, right?

Reply:
It is included now in requirements-dev.txt (for pytest -xv, python setup.py test runs fine without it)